src/main/java/seedu/address/model/entity/PrefixType.java

```diff
@@ -17,14 +17,12 @@ public String toStorageValue(){
     }
 
     public boolean isValidPrefix(String test){
-    try{
-        PrefixType result = PrefixType.valueOf(test);
-        return true;
-
-    } catch (IllegalArgumentException e){
-        return false;
-    }
-
+        try{
```

Comment:
is there a missing space here between try and the brace?

Reply:
oh yea lol. Dont worry i'll change it.